arm/Microsoft.Authorization/policyAssignments/.parameters/mg.parameters.json

```diff
@@ -41,7 +41,7 @@
         },
         "notScopes": {
             "value": [
-                "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg"
+                "/subscriptions/<<subscriptionId>>/resourceGroups/<<resourceGroupName>>"
```

Comment:
Do we have this token? Haven't seen it used anywhere else

Reply:
removed token